docs/en/observability/synthetics-analyze.asciidoc

```diff
@@ -0,0 +1,353 @@
+https://web.dev/vitals/[Core Web Vitals], an initiative that introduces a set of metrics
+that help categorize good and bad sites by quantifying the real-world user experience.
+====
+
+Next to each metric, there's an icon that indicates whether the value is
+higher (image:images/icons/sortUp.svg[Value is higher icon]),
+lower (image:images/icons/sortDown.svg[Value is lower icon]),
+or the same (image:images/icons/minus.svg[Value is the same])
+compared to all runs over the last 24 hours.
+
+// Screenshot of the viz
+[role="screenshot"]
+image::images/synthetics-analyze-one-step-metrics.png[Metrics visualization for one step in a browser monitor in the Synthetics UI]
+
+[discrete]
+[[synthetics-analyze-one-step-object]]
+=== Object weight and count
+
+++++
+<div style="color:#F04E98">
+++++
+
+// What info it includes
+The *Object weight* and *Object count* visualization shows...
+
+// What you might do with this info
+Use this information to...
+
+++++
+</div>
+++++
+
+// Screenshot of the viz
+[role="screenshot"]
+image::images/synthetics-analyze-one-step-object.png[Object visualization for one step in a browser monitor in the Synthetics UI]
+
+[discrete]
+[[synthetics-analyze-one-step-network]]
+=== Network requests
+
+// What info it includes
+The *Network requests* visualization is a waterfall chart that shows every request
+the page made when a user executed it.
+Each line in the chart represents an HTTP network request and helps you quickly identify
+what resources are taking the longest to load and in what order they are loading.
+
+The colored bars within each line indicate the time spent per resource.
+Each color represents a different part of that resource's loading process
+(as defined in the <<synthetics-analyze-one-step-timing>> section above) and
+includes the time spent downloading content for specific
+Multipurpose Internet Mail Extensions (MIME) types:
+HTML, JS, CSS, Media, Font, XHR, and Other.
+
+It's important to understand each phase of a request so you can improve your site's speed by
+reducing the time spent in each one.
+
+// Screenshot of the viz
+[role="screenshot"]
+image::images/synthetics-analyze-one-step-network.png[Network requests waterfall visualization for one step in a browser monitor in the Synthetics UI]
+
+Without leaving the waterfall chart, you can view data points relating to each resource:
+resource details, request headers, response headers, and certificate headers.
+On the waterfall chart, select a resource name, or any part of each row,
+to display the resource details overlay.
+
+For additional analysis, whether to check the content of a CSS file or to view a specific image,
+click image:images/icons/popout.svg[External link icon] located beside each resource,
+to view its content in a new tab.
+
+You can also navigate between steps and checks at the top of the page to
+view the corresponding waterfall charts.
+
+// [discrete]
+// [[synthetics-analyze-anomalies]]
+// = Anomalies
```

Comment:
I skipped this because it wasn't working when I wrote the initial draft, but it might be fixed now (https://github.com/elastic/kibana/issues/152384). I'll take another look.